components/match2/wikis/wildrift/match_summary.lua

```diff
@@ -0,0 +1,364 @@
+			championsDataIsEmpty = false
+		end
+		if String.isNotEmpty(extradata['team2champion' .. champIndex]) then
+			championsData[2][champIndex] = extradata['team2champion' .. champIndex]
+			championsDataIsEmpty = false
+		end
+		championsData[1].color = extradata.team1side
+		championsData[2].color = extradata.team2side
+	end
+
+	if
+		String.isEmpty(game.length) and
+		String.isEmpty(game.winner) and
+		championsDataIsEmpty
+	then
+		return nil
+	end
+
+	row	:addClass('brkts-popup-body-game')
+		:css('font-size', '85%')
+		:css('overflow', 'hidden')
+
+	row:addElement(CustomMatchSummary._opponentChampionsDisplay(championsData[1], _NUM_CHAMPIONS_PICK, false))
+	row:addElement(CustomMatchSummary._createCheckMark(game.winner == 1))
+	row:addElement(mw.html.create('div')
+		:wikitext(CustomMatchSummary._createAbbreviation{
+			title = String.isEmpty(game.length) and ('Game ' .. gameIndex .. ' picks') or 'Match Length',
+			text = game.length or ('Game ' .. gameIndex),
+		})
+	)
+	row:addElement(CustomMatchSummary._createCheckMark(game.winner == 2))
+	row:addElement(CustomMatchSummary._opponentChampionsDisplay(championsData[2], _NUM_CHAMPIONS_PICK, true))
+
+	-- Add Comment
+	if not Logic.isEmpty(game.comment) then
+		row:addElement(MatchSummary.Break():create())
+		local comment = mw.html.create('div')
+		comment :wikitext(game.comment)
+				:css('margin', 'auto')
+		row:addElement(comment)
+	end
+
+	return row
+end
+
+function CustomMatchSummary._createCheckMark(isWinner)
+	local container = mw.html.create('div')
+		:addClass('brkts-popup-spaced')
+		:css('line-height', '27px')
+		:css('margin-left', '3%')
+		:css('margin-right', '3%')
+
+	if isWinner then
+		container:node(_GREEN_CHECK)
+	else
+		container:node(_NO_CHECK)
+	end
+
+	return container
+end
+
+function CustomMatchSummary._createOpponent(opponent, side)
+	return OpponentDisplay.BlockOpponent{
+		flip = side == 'left',
+		opponent = opponent,
+		overflow = 'wrap',
+		teamStyle = 'short',
+	}
+end
+
+function CustomMatchSummary._createScore(opponent)
+	return OpponentDisplay.BlockScore{
+		isWinner = opponent.placement == 1 or opponent.advances,
+		scoreText = OpponentDisplay.InlineScore(opponent),
+	}
+end
+
+function CustomMatchSummary._createAbbreviation(args)
+	return '<i><abbr title="' .. args.title .. '">' .. args.text .. '</abbr></i>'
+end
+
+function CustomMatchSummary._opponentChampionsDisplay(opponentChampionsData, numberOfChampions, flip, isBan)
+	local opponentChampionsDisplay = {}
+	local color = opponentChampionsData.color or ''
+	opponentChampionsData.color = nil
+
+	for index = 1, numberOfChampions do
+		local champDisplay = mw.html.create('div')
+		:addClass('brkts-popup-side-color-' .. color)
+		:css('padding-left', '1px')--remove after css cache is gone
+		:css('padding-right', '1px')--remove after css cache is gone
+		:css('padding-top', '2px')--remove after css cache is gone
+		:css('padding-bottom', '2px')--remove after css cache is gone
```

Comment:
is css cache gone? (when was it added in the css)

Reply:
was a few days ago and since we will have to wait for another PR due to the above mentioned it will take another doy or more until this one here gets merged, all good then^^